README.md

```diff
@@ -1,58 +1,52 @@
-Welcome to the SCL.org Space
-============================
-This is basically an overview around the sclorg organization, aka what all one can find here.
+# Welcome to the SCL.org Space
 
-Why SCL.org? Since all the repositories, under this organization are somehow connected to the Software Collections concept. Software Collections give you the power to build, install, and use multiple versions of software on the same system, without affecting system-wide installed packages.
+This is an overview of the sclorg organization, aka all one can find here.
```

Comment:
Please do not use 'aka'. Use the full names.

Reply:
I rewrote some sentences to make them sound more professional :smile: